ql/src/java/org/apache/hadoop/hive/ql/ddl/table/info/desc/DescTableOperation.java

```diff
@@ -213,11 +214,14 @@ private void getColumnDataColPathSpecified(Table table, Partition part, List<Fie
       }
     } else {
       List<String> partitions = new ArrayList<String>();
-      partitions.add(part.getName());
+      // The partition name is converted to lowercase before generating the stats. So we should use the same
```

Comment:
Isn't this an issue?

I mean the `key=US` an `key=us` should be a different partition key, so modifying the values could cause issues later

Reply:
The lower case conversion is only for column name ..the value part is not converted.